corehq/apps/domain/templates/domain/admin/dhis2/dataset_map.html

```diff
@@ -4,16 +4,48 @@
 {% load crispy_forms_tags %}
 {% load i18n %}
 
+{% block js-inline %}{{ block.super }}
+    {% include 'domain/admin/partials/dhis2_dataset_map_templates.html' %}
+    <script src="{% static 'domain/js/dhis2_map_settings.js' %}"></script>
+    <script>
+    (function () {
```

Comment:
I'm not sure. I originally thought I should use `$(document).ready()`, but then realised that is not a thing any more, and noticed this pattern in [quite][1] [a][2] [few][3] [places][4]. 

But now that I search for `$(function...)` I'm seeing it everywhere, so I'll change this. 

I understand `$(function...)` is jQuery shorthand for `$(document).ready(function...)` and `(function...)()` just calls the function, but when should I use the second format? (Or are some of those instances I linked to above just the same mistake I made? :slightly_smiling_face: ) ... 

... After some googling, it seems at least some of those instances are definitely mistakes. (And the others are probably just logical collections of commands to be executed immediately, which is fair enough.)

  [1]: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/data_interfaces/templates/data_interfaces/interfaces/archive_forms.html#L23
  [2]: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/templates/domain/billing_statements.html#L12
  [3]: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/data_interfaces/templates/data_interfaces/interfaces/archive_forms.html#L23
  [4]: https://github.com/dimagi/commcare-hq/blob/master/custom/m4change/templates/m4change/selectTemplate.html#L14